packages/metals-vscode/src/extension.ts

```diff
@@ -1299,3 +1306,26 @@ function toggleBooleanWorkspaceSetting(setting: string) {
   const currentValues = configProperty?.workspaceValue ?? false;
   config.update(setting, !currentValues, ConfigurationTarget.Workspace);
 }
+
+function registerDebugEventListener(context: ExtensionContext) {
+  context.subscriptions.push(
+    debug.onDidReceiveDebugSessionCustomEvent((customEvent) => {
+      if (
+        customEvent.session.type !== SCALA_LANGID &&
```

Comment:
Why `!==` ? Shouldn't it be equal to "scala" ? Or does the debug adapter send in different type than "scala" ?

Reply:
It's a mistake, thanks for catching it :)